html-forms/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <div>
+      <label for="email">
+        <p>Email</p>
+        <input id="email" type="email" />
+      </label>
+    </div>
+    <div>
+      <label for="select">
+        <p>Why is coding cool?</p>
+        <select id="select">
+          <option value="option">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </label>
+    </div>
+    <div>
+      <div>
+        <label for="fav-language">
```

Comment:
This statement isn't labelling a specific form control element, so it doesn't need to be a label, you can just keep the `p` element